src/components/popup/tx-item.tsx

```diff
@@ -88,8 +90,41 @@ const Status: React.FC<{ transaction: Tx } & BoxProps> = ({ transaction, ...rest
   ) : null;
 };
 
+const SpeedUpButton = ({
+  txid,
+  isHovered,
+  isEnabled,
+}: {
+  txid: string;
+  isHovered: boolean;
+  isEnabled: boolean;
+}) => {
+  const [rawTxId, setTxId] = useRawTxIdState();
```

Comment:
I think later you use `setRawTxId` so maybe keep same here?

Reply:
will make changes in follow commit, going to merge!